Gaijinizer patch/Map191.txt

```diff
@@ -407,10 +440,10 @@ It was too big.#MTLed
 どんなに\\N[1]君が悪魔と接し、殺したとしても
 この器で暴走が始まるのは、数十年は先になるからね…
 > CONTEXT: Map191/events/1/pages/0/298/Dialogue/other2
-I was troubled. No matter how \\N[1] you deal
-with demon and kill you. It will be several
-decades before the runaway starts with this
-vessel ...#MTLed
+It was troubling. No matter how many demons
+\\N[1] ran into, even if he killed them all, it
+would take decades for the magic to run wild in
+that vessel of his...#RonIffyWording
 > END STRING
```

Comment:
Minor rewording:
>_**It was troubling. Even if \\\N[1] were to kill
every single demon he met, it would take
decades for the magic to run wild in that
vessel of his...**_